mscore/timeline.cpp

```diff
@@ -798,7 +798,7 @@ Timeline::Timeline(TDockWidget* dockWidget, QWidget* parent)
       std::tuple<int, qreal, Element*, Element*, bool> ri(0, 0, nullptr, nullptr, false);
       _repeatInfo = ri;
 
-      static const char* leftRepeat[] = {
+      static const char* startRepeat[] = {
```

Comment:
Wow this is a really weird way of hardcoding things. Interesting though ;)

Reply:
Not mine though, @dmitrio95's. We might instead use the SMuFL glyphs, like we do for Coda and Segno maybe

Reply:
No, not mine either :) Looks like originating from the initial timelime implementation, https://github.com/musescore/MuseScore/commit/2d5a6b022b52027501707ac57abc05693e972493

Reply:
Ah, darn, I confused it with your GSoC project, score-diff